### ISSUE TITLE
[PARConnector] PASolve and other functions are not exported in the package

### ISSUE THREAD

**Issue by fviale** [ow2-proactive/connector-r#78]
Maybe a recent R language change implied this issue.
```
> job <- PASolve("cos",1:5)
Error: could not find function "PASolve"
> job <- PARConnector:::PASolve("cos", 1:5)
Error in do.call("PA", dots, envir = parent.frame()) : 
  could not find function "PA"
```

**Comment by fviale** [comment]
The NAMESPACE file procuded by doxygen inside the 6.3.3 PARConnector package contains the following:
```
# Generated by roxygen2: do not edit by hand

export(PAConnect)
export(PADebug)
export(PADeleteFile)
export(PAGetResult)
export(PAJobState)
export(PAPullFile)
export(PAPushFile)
export(PAState)
export(PAWaitAny)
export(PAWaitFor)
exportClasses(PAJobResult)

import(rJava, stringr, codetools, devtools, gtools)

```

If the package is build from the master, the missing functions appear (PASolve, PA, etc)

**Comment by fviale** [comment]
As well, the output of the check task in jenkins is different than the one when done locally on windows:


Appenrently the documentation is not produced properly. The local build says "# Generated by roxygen2 (4.1.0): do not edit by hand", where the build on jenkins says "# Generated by roxygen2: do not edit by hand". Maybe there is an issue with the version of roxygen on the jenkins slave

Jenkins
```
:pa-rconnector:checkRPackage
Executing task ':pa-rconnector:checkRPackage' (up-to-date check took 0.038 secs) due to:
  No history is available.
Starting process 'command 'R''. Working directory: /home/activeeon/jenkins/workspace/connector-r 6.3/pa-rconnector/build Command: R CMD check --no-codoc --no-manual --no-multiarch /home/activeeon/jenkins/workspace/connector-r 6.3/pa-rconnector/build/r-pkg-src
Successfully started process 'command 'R''
* using log directory ‘/home/activeeon/jenkins/workspace/connector-r 6.3/pa-rconnector/build/r-pkg-src.Rcheck’
* using R version 3.2.2 (2015-08-14)
* using platform: x86_64-pc-linux-gnu (64-bit)
* using session charset: UTF-8
* using options ‘--no-codoc --no-manual’
* checking for file ‘r-pkg-src/DESCRIPTION’ ... OK
* this is package ‘PARConnector’ version ‘6.3.0’
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for executable files ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘PARConnector’ can be installed ... OK
* checking installed package size ... NOTE
  installed size is 21.5Mb
  sub-directories of 1Mb or more:
    java  21.0Mb
* checking package directory ... OK
* checking DESCRIPTION meta-information ... NOTE
Malformed Description field: should contain one or more complete sentences.
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking loading without being on the library search path ... OK
* checking dependencies in R code ... NOTE
Package in Depends field not imported from: ‘methods’
  These packages need to be imported from (in the NAMESPACE file)
  for when this namespace is loaded but not attached.
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... NOTE
js_print_job_data: warning in write.table(matrix, "", quote = F,
  row.name = F, col.names = F): partial argument match of 'row.name' to
  'row.names'
print_job_data: warning in write.table(matrix, "", quote = F, row.name
  = F, col.names = c("ID", "NAME", "OWNER", "PRIORITY", "STATUS",
  "START_AT", "DURATION")): partial argument match of 'row.name' to
  'row.names'
print_task_data: warning in write.table(matrix, "", quote = F, row.name
  = F, col.names = c("ID", "NAME", "ITER", "DUP", "STATUS", "HOSTNAME",
  "EXEC DURATION", "TOT DURATION", "#NODES USED", "#EXECUTIONS",
  "#NODES KILLED")): partial argument match of 'row.name' to
  'row.names'
.createAndTransferPAFileFromPattern: no visible binding for global
  variable ‘client’
PAConnect: no visible global function definition for
  ‘.rs.askForPassword’
.unserializeObj: no visible global function definition for ‘eng.assign’
.unserializeObj: no visible binding for global variable ‘tmpoutput’
print,PAJobResult: no visible binding for global variable ‘object’
* checking Rd files ... WARNING
checkRd: (7) PAConnect.Rd:23-26: Tag \dontrun not recognized
checkRd: (7) PADeleteFile.Rd:23-25: Tag \dontrun not recognized
checkRd: (7) PAM.Rd:43-60: Tag \dontrun not recognized
checkRd: (7) PAPullFile.Rd:27-29: Tag \dontrun not recognized
checkRd: (7) PAPushFile.Rd:27-29: Tag \dontrun not recognized
checkRd: (7) PA.Rd:120-137: Tag \dontrun not recognized
checkRd: (7) PASolve.Rd:28-85: Tag \dontrun not recognized
checkRd: (7) PAS.Rd:42-57: Tag \dontrun not recognized
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... WARNING
Undocumented S4 methods:
  generic 'PAWaitAny' and siglist 'PAJobResultOrMissing'
  generic 'PAWaitFor' and siglist 'PAJobResultOrMissing'
All user-level objects in a package (including S4 classes and methods)
should have documentation entries.
See chapter ‘Writing R documentation files’ in the ‘Writing R
Extensions’ manual.
* checking for code/documentation mismatches ... SKIPPED
* checking Rd \usage sections ... WARNING
Undocumented arguments in documentation object 'PA'
  ‘funcOrFuncName’ ‘...’ ‘varies’ ‘input.files’ ‘output.files’ ‘in.dir’
  ‘out.dir’ ‘hostname.selection’ ‘ip.selection’
  ‘property.selection.name’ ‘property.selection.value’ ‘nodes.number’
  ‘topology’ ‘generic.information.list’ ‘run.as.me’ ‘walltime’
  ‘isolate.io.files’ ‘client’ ‘.debug’

Undocumented arguments in documentation object 'PAConnect'
  ‘url’ ‘login’ ‘pwd’ ‘cred’ ‘insecure’ ‘.print.stack’

Undocumented arguments in documentation object 'PADeleteFile'
  ‘space’ ‘pathname’ ‘client’

Undocumented arguments in documentation object 'PAM'
  ‘funcOrFuncName’ ‘...’ ‘varies’ ‘input.files’ ‘output.files’ ‘in.dir’
  ‘out.dir’ ‘hostname.selection’ ‘ip.selection’
  ‘property.selection.name’ ‘property.selection.value’ ‘nodes.number’
  ‘topology’ ‘generic.information.list’ ‘run.as.me’ ‘walltime’
  ‘isolate.io.files’ ‘client’ ‘.debug’

Undocumented arguments in documentation object 'PAPullFile'
  ‘space’ ‘pathname’ ‘outputFile’ ‘client’ ‘.nb.tries’ ‘.print.stack’

Undocumented arguments in documentation object 'PAPushFile'
  ‘space’ ‘path’ ‘fileName’ ‘inputFile’ ‘client’ ‘.print.stack’

Undocumented arguments in documentation object 'PAS'
  ‘funcOrFuncName’ ‘...’ ‘varies’ ‘input.files’ ‘output.files’ ‘in.dir’
  ‘out.dir’ ‘hostname.selection’ ‘ip.selection’
  ‘property.selection.name’ ‘property.selection.value’ ‘nodes.number’
  ‘topology’ ‘generic.information.list’ ‘run.as.me’ ‘walltime’
  ‘isolate.io.files’ ‘client’ ‘.debug’

Undocumented arguments in documentation object 'PASolve'
  ‘...’ ‘client’ ‘.debug’ ‘jobName’ ‘jobDescription’ ‘priority’
  ‘cancelOnError’

Documented arguments not in \usage in documentation object 'PAWaitAny':
  ‘timeout’ ‘client’ ‘callback’

Documented arguments not in \usage in documentation object 'PAWaitFor':
  ‘timeout’ ‘client’ ‘callback’

Undocumented arguments in documentation object '[[,PAJobResult-method'
  ‘x’ ‘i’ ‘j’ ‘drop’

Functions with \usage entries need to have the appropriate \alias
entries, and all their arguments documented.
The \usage entries must correspond to syntactically valid R code.
See chapter ‘Writing R documentation files’ in the ‘Writing R
Extensions’ manual.
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking installed files from ‘inst/doc’ ... OK
* checking examples ... OK
* DONE

Status: 3 WARNINGs, 4 NOTEs
See
```

Local
```
:pa-rconnector:checkRPackage
* using log directory 'C:/GitHub/connector-r/pa-rconnector/build/r-pkg-src.Rcheck'
* using R version 3.2.3 (2015-12-10)
* using platform: x86_64-w64-mingw32 (64-bit)
* using session charset: ISO8859-1
* using options '--no-codoc --no-manual'
* checking for file 'r-pkg-src/DESCRIPTION' ... OK
* this is package 'PARConnector' version '6.3.0'
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for executable files ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking whether package 'PARConnector' can be installed ... OK
* checking installed package size ... NOTE
  installed size is 21.5Mb
  sub-directories of 1Mb or more:
    java  21.1Mb
* checking package directory ... OK
* checking DESCRIPTION meta-information ... NOTE
Malformed Description field: should contain one or more complete sentences.
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking loading without being on the library search path ... OK
* checking dependencies in R code ... NOTE
Package in Depends field not imported from: 'methods'
  These packages need to be imported from (in the NAMESPACE file)
  for when this namespace is loaded but not attached.
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... NOTE
.createAndTransferPAFileFromPattern: no visible binding for global
  variable 'client'
.unserializeObj: no visible global function definition for 'eng.assign'
.unserializeObj: no visible binding for global variable 'tmpoutput'
PAConnect: no visible global function definition for
  '.rs.askForPassword'
js_print_job_data: warning in write.table(matrix, "", quote = F,
  row.name = F, col.names = F): correspondance partielle d'argument de
  'row.name' par rapport Ó 'row.names'
print_job_data: warning in write.table(matrix, "", quote = F, row.name
  = F, col.names = c("ID", "NAME", "OWNER", "PRIORITY", "STATUS",
  "START_AT", "DURATION")): correspondance partielle d'argument de
  'row.name' par rapport Ó 'row.names'
print_task_data: warning in write.table(matrix, "", quote = F, row.name
  = F, col.names = c("ID", "NAME", "ITER", "DUP", "STATUS", "HOSTNAME",
  "EXEC DURATION", "TOT DURATION", "#NODES USED", "#EXECUTIONS",
  "#NODES KILLED")): correspondance partielle d'argument de 'row.name'
  par rapport Ó 'row.names'
print,PAJobResult: no visible binding for global variable 'object'
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... SKIPPED
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking installed files from 'inst/doc' ... OK
* checking examples ... OK

* DONE
See
```

**Comment by fviale** [comment]
fixed by commit b27920f8c0ebe56326206b91f42a16de31357725
